### PR TITLE
ENH: Save coordinate system unit to markups json files

### PR DESCRIPTION
**Pull request by lassoan**
Also improve saving of units into markups measurements and make error message display during loading.

see https://github.com/Slicer/Slicer/issues/5261

### CODE REVIEW

File: Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.1.json [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+                                    "anyOf": [
+                                        {
+                                            "type": "string",
+                                            "title": "Units of control point coordinates",
+                                            "description": "Control point coordinate values are specified in this length unit. Specified in UCUM.",
+                                            "default": "mm",
+                                            "enum": ["mm", "um"]
+                                        },
+                                        {
+                                            ttt
```
Comment (Sunderlandkyl) [inline Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.1.json:66]: Remove?

Comment (lassoan) [reply Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.1.json:66]: Good catch thank you! It seems I've not committed the final version.

**Comment by lassoan** [comment]
I've made some simplifications in how units in measurements are specified (moved more into the measurement base class) and added one more test, @Sunderlandkyl please have a look. Thanks!

**Review by Sunderlandkyl — APPROVED** [review]
I've never really looked at the vtkMRMLMeasurement code, but it looks good to me.

**Pull request merged**